src/glibc/misc/fsync.c

```diff
@@ -18,12 +18,13 @@
 #include <errno.h>
 #include <unistd.h>
 #include <syscall-template.h>
+#include <lind_syscall_num.h>
 
 /* Make all changes done to FD actually appear on disk.  */
 int
 fsync (int fd)
 {
-	return MAKE_SYSCALL(162, "syscall|fsync", (uint64_t) fd, NOTUSED, NOTUSED, NOTUSED, NOTUSED, NOTUSED);
+	return MAKE_TRADITION(FSYNC_SYSCALL, "syscall|fsync", (uint64_t) fd, NOTUSED, NOTUSED, NOTUSED, NOTUSED, NOTUSED, WRAPPED_SYSCALL);
```

Comment:
More name nitpicking. I know you're just using code from before here but seems like a good opportunity for us to clarify. I never really understoood the Wrapped vs Raw difference. It seems just to be because of errno handling right? If thats true those names are really non-sensical to me. What if we changed it to a handle_errno flag or something like that?

Reply:
or like TRANSLATE_ERRNO is probably clearer